README.md

```diff
@@ -180,6 +180,8 @@ They exist to document what may be implemented by [Nostr](https://github.com/nos
 | `22242`       | Client Authentication           | [42](42.md)                            |
 | `23194`       | Wallet Request                  | [47](47.md)                            |
 | `23195`       | Wallet Response                 | [47](47.md)                            |
+| `23196`       | NWC Notification (v0)           | [47](47.md)                            |
+| `23197`       | NWC Notification (v1+)          | [47](47.md)                            |
```

Comment:
Would it also make sense to propose `v` as a standardized tag (further down in this file)?

Reply:
Better to stay out of it since NIP-47 seems to be the only place using it. Let's see if other nips decide to adopt first.